Assets/Scripts/Layers/LayerTypes/LayerData.cs

```diff
@@ -205,5 +205,22 @@ public void RemoveProperty(LayerPropertyData propertyData)
             layerProperties.Remove(propertyData);
             PropertyRemoved.Invoke(propertyData);
         }
+
+        /// <summary>
+        /// Recursively collect all assets from each of the property data elements for loading and saving
+        /// purposes. 
+        /// </summary>
+        /// <returns>A list of assets on disk</returns>
+        public IEnumerable<Asset> GetAssets()
```

Comment:
Hoort dit thuis in LayerData? ik vind het meer iets voor ReferencedLayerData omdat die assets (kan) hebben en normale LayerData niet

Reply:
Ik denk dat er mogelijk verwarring is rondom het woord Asset hier; Asset is geen unity asset maar een project asset - en dus een file die in de zip moet worden opgeslagen. In basis kan elke laag assets hebben vanuit diens properties